### PR TITLE
Bug 1853198: [release-4.3] Backport OVS-retries

### PR DESCRIPTION
**Pull request by alexanderConstantinescu**
4.3 backport of #153 and #159

/assign @danwinship @aojea

### CODE REVIEW

**Comment by alexanderConstantinescu** [comment]
@danwinship PTAL

File: pkg/network/node/metrics/metrics.go [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // +build linux
 
-package node
+package metrics
```
Comment (danwinship) [inline pkg/network/node/metrics/metrics.go:3]: To the extent possible, you should be backporting only the new change you made. So don't backport moving `metrics.go` into a different directory

File: pkg/network/node/metrics/metrics.go [hunk 2]
```diff
@@ -44,6 +49,15 @@ var (
 			Help:      "Number of Open vSwitch flows",
 		},
 	)
+	OVSOperationsResult = metrics.NewCounterVec(
+		&metrics.CounterOpts{
```
Comment (danwinship) [inline pkg/network/node/metrics/metrics.go:53]: This should be using `prometheus.`, not `metrics.`, like the other metrics in this file. (Apparently at some point the dependency got moved around or whatever? But you want to be using the same version of it as everything else in 4.3.)

File: pkg/network/node/metrics/metrics.go (hunk 1 above)
Comment (alexanderConstantinescu) [reply pkg/network/node/metrics/metrics.go:3]: All back-port included this exact patch:

https://github.com/openshift/sdn/pull/155/files
https://github.com/openshift/sdn/pull/154/files

So I am not sure what you're referring to?

**Comment by alexanderConstantinescu** [comment]
@danwinship : I've addressed your comment. PTAL

**Pull request merged**